pom.xml

```diff
@@ -352,5 +357,27 @@
         <javadoc.opts>-Xdoclint:none</javadoc.opts>
       </properties>
     </profile>
+    <profile>
```

Comment:
It's better to move this profile to integration-test pom.xml. Let it be cohesive.